test/OpenTelemetry.Tests/BatchTest.cs

```diff
@@ -18,7 +18,7 @@
 using OpenTelemetry.Internal;
 using Xunit;
 
-namespace OpenTelemetry.Tests
+namespace OpenTelemetry.Trace.Tests
```

Comment:
Should this file be moved to the `Trace` directory?

Reply:
yes, thanks!